File: src/SimpleBlog/ClientApp/src/app/core/constants/validation.constants.ts
export class ValidationConstants {
  public static readonly USERNAME_REG_EX = /^[a-z]{1}[a-z0-9|\-|_]{2,23}[a-z0-9]{1}$/i;
  public static readonly EMAIL_REG_EX = /^[^@\s]+@[^@\s]+$/i;
}

File: src/SimpleBlog/Constants/ValidationConstants.cs
using System.Text.RegularExpressions;

namespace SimpleBlog.Constants;

public static class ValidationConstants
{
    public static readonly Regex USERNAME = new Regex(@"^[a-z]{1}[a-z0-9|\-|_]{2,23}[a-z0-9]{1}$", RegexOptions.IgnoreCase);
    public static readonly Regex EMAIL = new Regex(@"^[^@\s]+@[^@\s]+$", RegexOptions.IgnoreCase);
}
